langchain4j/src/test/java/dev/langchain4j/service/AiServicesIT.java

```diff
@@ -1086,4 +1088,97 @@ void should_execute_multiple_tools_in_parallel_then_answer() {
                 singletonList(calculatorSpecification)
         );
     }
+
+    static class IdsProcessor {
+
+        @Tool("processes list of uuids")
+        String processUuids(@P("uuids to use") List<String> uuids) {
+            return String.join("_", uuids);
+        }
+    }
+
+    @Test
+    void should_use_tool_with_array_parameters_then_answer() {
+        ToolSpecification listProcessorSpecification = ToolSpecification.builder()
+            .name("processUuids")
+            .description("processes list of uuids")
+            .addParameter("arg0", ARRAY, JsonSchemaProperty.items(JsonSchemaProperty.STRING), JsonSchemaProperty.description("uuids to use"))
```

Comment:
```suggestion
            .addParameter("arg0", ARRAY, items(STRING), description("uuids to use"))
```

Please use static imports